qdrant-landing/content/blog/qdrant-1.12.x.md

```diff
@@ -0,0 +1,266 @@
+---
+title: "Qdrant 1.12 - Distance Matrix, Facet Counting & On-Disk Indexing"
+draft: false
+short_description: "On-Disk Text & Geo Index. Distance Matrix API. Facet API for Cardinality."
+description: "Uncover insights with the Distance Matrix API, dynamically filter via Facet API, and offload additional payload to disk." 
+preview_image: /blog/qdrant-1.12.x/social_preview.png
+social_preview_image: /blog/qdrant-1.12.x/social_preview.png
+date: 2024-10-03T00:00:00-08:00
+author: David Myriel
+featured: true
+tags:
+  - vector search
+  - distance matrix 
+  - dimensionality reduction
+  - data exploration
+  - data visualization
+  - faceting
+  - facet api
+---
+[**Qdrant 1.12.0 is out!**](https://github.com/qdrant/qdrant/releases/tag/v1.12.0) Let's look at major new features and a few minor additions:
+
+**Distance Matrix API:** Efficiently calculate pairwise distances between vectors.</br>
+**GUI Data Exploration** Visually navigate your dataset and analyze vector relationships.</br>
+**Faceting API:** Dynamically aggregate and count unique values in specific fields.</br>
+
+**Text Index on_disk:** Reduce memory usage by storing text indexing data on disk.</br>
+**Geo Index on_disk:** Offload indexed geographic data on disk for memory efficiency.
```

Comment:
While the parameter is named this way, I think it's a bit better readable without the underscore:

```suggestion
**Text Index on disk:** Reduce memory usage by storing text indexing data on disk.</br>
**Geo Index on disk:** Offload indexed geographic data on disk for memory efficiency.
```